src/utils/isCustomPropertySet.js

```diff
@@ -8,12 +7,13 @@ import hasBlock from "./hasBlock";
  * @returns {boolean}
  */
 export default function(node) {
-  const selector = get(node, "raws.selector.raw", node.selector);
```

Comment:
@kristerkari this had to be modified since there’s a difference between how custom properties are parsed in PostCSS 7 and 8. They’re declarations instead of rules. Can’t see anything in changelog though :/

Reply:
We ran into the same change in behaviour with migrating to PostCSS 8.

Your changes to the util LGTM.